app/controllers/shipit/shipit_controller.rb

```diff
@@ -66,5 +67,14 @@ def set_variant
       request.format = :html
       request.variant = :partial
     end
+
+    def check_github_status
+      github_status = Rails.cache.read('github::status')
```

Comment:
Since you have a `GithubStatus` namespace, that cache read should be encapsulated there as well.